changelog.md

```diff
@@ -36,6 +36,11 @@
   and this can now throw in edge cases where `getCurrentDir` throws.
   `relativePath` also now works for js with `-d:nodejs`.
 
+- `streams.StringStream` is now supported in JS, with the limitation that any
+  buffer `pointer`s used must semantically be `ptr string`, any pointer
+  incompatible with `ptr string` will not work. This should allow the following
+  standard library modules to compile on JS: `parsecsv`, `parsecfg`, `parsesql`,
```

Comment:
this definitely needs a test; create a new
`tests/js/tstdlib_imports.nim` that imports every module supported by js
analog to `tests/test_nimscript.nims` for js instead of nimscript, eg:

```nim
import strutils
import parsecsv
import parsecfg
...
```
if you feel like it, add every js module for which this works, if not, at least all the modules you've now enabled to work with it and add a `# xxx add other modules`